src/modules/database/db.providers.ts

```diff
@@ -0,0 +1,27 @@
+import { Sequelize } from 'sequelize-typescript';
+import { Options } from 'sequelize';
+import { environment } from 'src/environments/environment';
+import { Users } from '../user/models/users';
+import { UserProfile } from '../user/models/user-profile';
+import * as mock_data from '../../../test/MOCK_DATA';
+
+export const databaseProviders = [
```

Comment:
please provide users only during 1st run or prepare separate script that creates fake data

Reply:
Decided to create users in first run or update them instead.